### PR TITLE
Increase collection of benchmark artifacts to 30 for better averaging

### PR DESCRIPTION
**Pull request by agunapal**
## Description

The nightly benchmark run failed at auto-validation.
    - Increase window size to 30 to get a more accurate average
    - Temporarily turn off erroring to avoid alarms
    

Fixes #([issue](https://github.com/pytorch/serve/issues/2255))

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Feature/Issue validation/testing

N/A

## Checklist:

- [ ] Did you have fun?
- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Review by namannandan — APPROVED** [review]

File: benchmarks/validate_report.py [hunk 1]
```diff
@@ -63,9 +63,6 @@ def validate_reports(artifacts_dir, report_dir, deviation):
         if not error:
             print(f"Model {model} successfully validated")
 
-    if error:
-        raise Exception("Failures in benchmark validation")
-
```
Comment (min-jean-cho) [inline benchmarks/validate_report.py:68]: why we remove this?

Comment (agunapal) [reply benchmarks/validate_report.py:68]: Below are the failures from Benchmark CPU run.

Nothing seems totally wrong. However, we need a good way to differentiate between a one off failure vs something fundamentally broken.

Till we fix that, it is going to raise a github issue everyday. So, removing the raise Exception  will reduce the noise.

Hopefully with window size increasing to 30, we will see a more accurate average value. Once we are confident, then we can turn this back on.

```
Error while validating cpu_percentage_mean for model: eager_mode_vgg16_w4_b8, Expected value: 30.85, Observed value: 50.00
Error while validating TS throughput for model: scripted_mode_bert_w4_b1, Expected value: 17.51, Observed value: 36.03
Error while validating cpu_percentage_mean for model: scripted_mode_bert_w4_b1, Expected value: 34.06, Observed value: 80.00
Error while validating TS throughput for model: scripted_mode_bert_w4_b2, Expected value: 20.16, Observed value: 41.
Error while validating TS throughput for model: scripted_mode_bert_w4_b4, Expected value: 21.29, Observed value: 42.67
Error while validating cpu_percentage_mean for model: scripted_mode_bert_w4_b4, Expected value: 29.45, Observed value: 75.00
Error while validating TS throughput for model: scripted_mode_bert_w4_b8, Expected value: 22.78, Observed value: [42](https://github.com/pytorch/serve/actions/runs/4760803067/jobs/8461463678#step:12:43).44
Error while validating cpu_percentage_mean for model: scripted_mode_bert_w4_b8, Expected value: 27.55, Observed value: 50.00
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b1, Expected value: 13.94, Observed value: 24.31
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b2, Expected value: 16.55, Observed value: 27.25
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b4, Expected value: 17.27, Observed value: 27.13
Error while validating cpu_percentage_mean for model: scripted_mode_vgg16_w4_b4, Expected value: 30.19, Observed value: 55.10
Error while validating TS throughput for model: scripted_mode_vgg16_w4_b8, Expected value: 18.08, Observed value: 26.67
```

Comment (min-jean-cho) [reply benchmarks/validate_report.py:68]: I see, thanks -- makes sense.

**Review by min-jean-cho — APPROVED** [review]

File: benchmarks/validate_report.py (hunk 1 above)
Comment (agunapal) [reply benchmarks/validate_report.py:68]: I noticed that the throughput failures above are not valid. If throughput is increasing, then it should be a don't care from a validation perspective. Pushed a small change to fix that.

**Pull request merged**